ush/config.sh.RRFS_CONUS_3km_ens

```diff
@@ -75,46 +69,57 @@ ENDDAY=${DATE_LAST_CYCL:6:2}
 ENDHOUR="23"
 
 PREEXISTING_DIR_METHOD="upgrade" # "rename"
-INITIAL_CYCLEDEF="${DATE_FIRST_CYCL}1800 ${DATE_LAST_CYCL}2300 24:00:00"
+INITIAL_CYCLEDEF="${DATE_FIRST_CYCL}1800 ${DATE_LAST_CYCL}2300 12:00:00"
 BOUNDARY_CYCLEDEF="${DATE_FIRST_CYCL}1800 ${DATE_LAST_CYCL}2300 06:00:00"
-PROD_CYCLEDEF="${DATE_FIRST_CYCL}1900 ${DATE_LAST_CYCL}2300 01:00:00"
+PROD_CYCLEDEF="${DATE_FIRST_CYCL}1800 ${DATE_LAST_CYCL}2300 01:00:00"
```

Comment:
This one should be 1 hour after the first spin-up cycle, ${DATE_FIRST_CYCL}1900, since we have INITIAL_CYCLEDEF="${DATE_FIRST_CYCL}1800

Reply:
changed to start from 19Z.